src/rest.rs

```diff
@@ -91,10 +89,8 @@ impl From<Transaction> for TransactionValue {
             txid: tx.txid(),
             vin,
             vout,
-            confirmations: None,
-            hex: None,
-            block_hash: None,
             size: bytes.len() as u32,
+            hex: hex::encode(bytes),
```

Comment:
If I remember correctly, hex was an option and not inlcuded in the blocks view to save space, while included in the transaction view